mbq/atomiq/consumers.py

```diff
@@ -40,46 +30,18 @@ def process_task(self, task):
                 task.failed_at = arrow.utcnow().datetime
                 task.error_message = str(e)
                 task.stacktrace = traceback.format_exc()
-                rollbar.report_exc_info()
```

Comment:
how come we don't want to rollbar here? doesn't sending the error to rollbar just make it easier to debug why a task failed?

Reply:
we already have a monitor for failed tasks, we have logs, we're storing error message and stack trace, it's a handled failure, I think rollbar is unnecessary.